pyramid_swagger/tween.py

```diff
@@ -309,16 +310,14 @@ def json(self, **kwargs):
         return getattr(self.response, 'json_body', {})
 
 
-def handle_request(request, validator_map, validation_context, **kwargs):
+def handle_request(request, validator_map, **kwargs):
     """Validate the request against the swagger spec and return a dict with
     all parameter values available in the request, casted to the expected
     python type.
 
     :param request: a :class:`PyramidSwaggerRequest` to validate
     :param validator_map: a :class:`pyramid_swagger.load_schema.ValidatorMap`
         used to validate the request
-    :param validation_context: a context manager for wrapping validation
-        errors
     :returns: a :class:`dict` of request data for each parameter in the swagger
```

Comment:
Can you add a :raises: param here documenting that it returns the consolidated RequestValidationError?